modules/distribution-zones/src/integrationTest/java/org/apache/ignite/internal/rebalance/ItRebalanceDistributedTest.java

```diff
@@ -1528,7 +1528,10 @@ private class Node {
                             placementDriver,
                             schemaSyncService,
                             systemDistributedConfiguration,
-                            sharedTxStateStorage
+                            sharedTxStateStorage,
+                            txManager,
+                            catalogManager,
+                            schemaManager
```

Comment:
As far as I can see schemaManager is used in order to `waitForSchemaAvailability` in `CatalogValidationSchemasSource`, however we already have `schemaSyncService` in `PartitionReplicaListener` with `waitForMetadataCompleteness`. I didn't go deeper in order to understand whether it's the same or not, however from the naming it seems that it's somewhat relative. Do we really need schemaManager?

Reply:
`waitForSchemaAvailability()` is used to wait for a specific table version to become available, it uses schema registry and not schema sync service, so it's not the same thing